### DISCUSSION TITLE
do another command after nuxt generate

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [nuxt/nuxt#27013] [Questions]
Hi all,
I noticed when 'npm run generate' is finished, it shows the success message "You can now deploy..." but it does not exit. I would like to run another script after the generate, so something like...

`nuxt generate && other command`

But because the first one hangs and does not end. the next one is never run.
Has anyone had any success chaining commands in the NPM run script?

**Comment by AloisSeckar** [comment]
There is no general problem in chaining npm-based comands in package.json's scripts. I even have sort of a "pipeline" for linting, testing, auto-increment version and deploying it into package registry all at once.

For sure there might be commands like `nuxt dev` that start some service and keep blocking the terminal, but `nuxt generate` shoudn't "hang" like that- it should perform a static build and end. At least my works so. Just tried `nuxt generate && npm install` and both were executed.

What is your console output?
